string id
string time_ref
float32 x
float32 y
float32 a
---
bool success
string reason
float32 velocity
float32 steering